modules/ROOT/pages/authentication-authorization/limitations.adoc

```diff
@@ -277,6 +277,63 @@ CALL db.labels()
 
 will only return label `:A`, because that is the only label for which traversal was granted.
 
+[[access-control-limitations-non-existing-labels]]
+=== Privileges and non-existing labels, relationship types and property names
+
+Privileges on non-existing labels, relationship types, or property names will not take effect.
```

Comment:
Maybe add something like `until they are created`. It would be nice to clarify to avoid leading the user into the assumption that the privilege will never work if the label did not exists when the privilege was created. I know this get's explained later on in the example, but just wan't to avoid misunderstandings early on.